backend/handlers/handlers.go

```diff
@@ -765,11 +770,38 @@ func (handlers *Handlers) postExportAccountSummary(_ *http.Request) (interface{}
 		accountName := account.Name()
 		balance := account.Balance().Available().BigInt().String()
 		unit := account.Coin().SmallestUnit()
+		var accountType string
+		var xpubs string
+		var address string
+		var scriptType string
+		if signingConfiguration := account.Info().SigningConfiguration; signingConfiguration.IsAddressBased() {
+			accountType = "address"
+			address = signingConfiguration.Address()
+			scriptType = ""
+		} else {
```

Comment:
In case we have both the address and the xpub, shouldn't we add both? Thus instead of `else` have an `if` the xpub is set.

Reply:
Yes, I had it so both were added, hmm, not sure what happened there :)

Reply:
Having the address in the signing configuration will change how this is done anyway

Reply:
Resolved by making an account AddressBased only if it has an address AND no xpub on configuration.go 
`return configuration.address != "" && len(configuration.ExtendedPublicKeys()) == 0`